src/process.rs

```diff
@@ -500,19 +543,19 @@ where
 ///
 /// The first `i32` value is a pointer
 #[derive(serde::Serialize, serde::Deserialize)]
-enum Sendable {
+pub enum Sendable<S = Bincode> {
```

Comment:
I think the `S` type is unnecessary here, this information is erased between processes and is already known in the `send` method.

Reply:
Seems like I was able to remove it just fine :+1: